langfuse-langchain/src/callback.ts

```diff
@@ -488,16 +495,34 @@ export class CallbackHandler extends BaseCallbackHandler {
     }
   }
 
+  /**
+   * Merges tags and metadata into a single object.
+   *
+   * This method combines an array of tags and two metadata objects into a single
+   * metadata object. If tags are provided, they are added under the key 'tags'.
+   * The two metadata objects are merged into this final object, with values from
+   * `metadata2` potentially overwriting those from `metadata1` if they share keys.
+   *
+   * @param {string[] | undefined} tags - An optional array of tags to include.
+   * @param {Record<string, unknown> | undefined} metadata1 - The first optional metadata object.
+   * @param {Record<string, unknown> | undefined} metadata2 - The second optional metadata object.
+   * @returns {Record<string, unknown>} The merged metadata object containing tags and both metadata objects.
```

Comment:
Wondering how much time we should put into these docs. A few sentences are good for sure but all these parameters etc. need to be maintained.